src/webgl/light.js

```diff
@@ -592,13 +592,63 @@ p5.prototype.pointLight = function(v1, v2, v3, x, y, z) {
  * @alt
  * light with slider having a slider for varying roughness
  */
-p5.prototype.imageLight = function(img){
+p5.prototype.imageLight = function (img) {
   // activeImageLight property is checked by _setFillUniforms
   // for sending uniforms to the fillshader
   this._renderer.activeImageLight = img;
   this._renderer._enableLighting = true;
 };
 
+/**
+ * Creates a Panorama with given image.
+ *
+ *
+ * `panorama(img)` is a method designed to transform a standard
+ * image into a 360-degree view. It operates on the concept
+ * of sphere mapping, where the image is manipulated to
+ * resemble a sphere by adjusting camera angles. Utilizing
+ * this method, users can obtain a complete 360-degree view
+ * of a scene.
+ *
+ * Using Panorama is straightforward. Similar to calling a
+ * `background(color)`, users only need to call the `panorama(img)`, and
+ * beneath it, anything created will form a 360-degree scene.
+ * To enable 360-degree viewing, it is essential to invoke
+ * `orbitControl()`; otherwise, the method will not function as intended.
```

Comment:
Maybe instead of saying it's essential, since it still works if you programmatically animate a camera, we can say to either use `orbitControl` or [try changing the orientation of the camera](https://p5js.org/reference/#/p5/camera) to see different parts of the background.